### PR TITLE
Add Calico Enterprise marketing doc

### PR DESCRIPTION
**Pull request by bmckercher123**
Part of new initiative by Andy and Alex to add CE upsell content to OS:

- Merge ASAP 
- Reviewed by Alex and Andy
- Cross link in "Adopt zero trust" and "Get started with Calico network policy" and "Kubernetes network policy"
- Known side effect that folder will automatically merge into TSEE docs (Alex P)
- Product name is hardcoded to differentiate between OS and CE (until we get new variables)

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 86324a24c22044434f1134b8da3f7632fbe2fe50

https://deploy-preview-3061--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 43efb01a1a81fd0203d36c190e9d320e41a3a85d

https://deploy-preview-3061--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit c045d5cf176fd02f84b4549c549873e6c52446a7

https://deploy-preview-3061--calico.netlify.com

File: _data/master/navbars/security.yml [hunk 1]
```diff
@@ -4,103 +4,75 @@ order: 3
...
+      path: /security/network-policy-openstack   
   - title: Kubernetes policy
     section: 
-
     - title: Get started with Kubernetes network policy
       path: /security/kubernetes-network-policy
-
     - title: Enable default deny for Kubernetes pods
       path: /security/kubernetes-default-deny 
-
```
Comment (caseydavenport) [inline _data/master/navbars/security.yml]: Could you undo these whitespace changes?

**Comment by lxpollitt** [comment]
Deploy preview has some issues currently.  e.g. The link from the left nav for the new page is https://deploy-preview-3061--calico.netlify.com/master/security/calico-enterprise/network-visibility is a 404

**Pull request merged**